app/views/ad_templates/default/show.html.erb

```diff
@@ -2,16 +2,18 @@
   <div id="cf">
     <span>
       <span class="cf-wrapper">
-        <a data-href="campaign_url" class="cf-img-wrapper" target="_blank" rel="nofollow noopener">
-          <%= image_tag @creative.large_image.cloudfront_url, border: 0, height: 100, width: 130, class: "cf-img" rescue nil %>
-        </a>
-        <a data-href="campaign_url" class="cf-text" target="_blank" rel="nofollow noopener">
-          <strong><%= @creative.headline %></strong>
-          <span><%= @creative.body %></span>
+        <a data-href="campaign_url" target="_blank" rel="nofollow noopener">
+          <span class="cf-img-wrapper">
+            <%= image_tag @creative.large_image.cloudfront_url, border: 0, height: 100, width: 130, class: "cf-img", alt: @creative.name rescue nil %>
+          </span>
+          <span class="cf-text">
+            <strong><%= @creative.headline %></strong>
+            <span><%= @creative.body %></span>
+          </span>
```

Comment:
Are the new wrapper tags serving a functional purpose here? More stylistic control perhaps?

Reply:
The fix the redundant link issue.